app/Services/DivisionApi/Adapters/EUD.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Services\DivisionApi\Adapters;
+
+use App\Contracts\DivisionApiContract;
+use App\Models\Area;
+use App\Models\User;
+use Illuminate\Support\Facades\Http;
+
+class EUD implements DivisionApiContract
```

Comment:
question: how 'bout VATEUD to use full names?

Reply:
Yeah that would maybe make more sense